DRAFT: Initial files for 2023 TOC Elections

geekygirldawn opened the pull request:
Here is a PR with the initial files required to kick off the Knative 2023 TOC elections.

Before this PR can be merged:
* @jberkus needs to confirm that he can have everything needed to set up Elekto done by May 2.
* @knative/steering-committee needs to officially appoint the election officers. We've proposed @geekygirldawn and @aliok.
* @knative/steering-committee needs to make sure they are OK with the timeline and other details proposed in this PR.

/hold

geekygirldawn commented:
Note: I've left @jberkus in the OWNERS file deliberately in case there are any urgent issues, especially related to Elekto, that might come up when the Election Officers are offline (esp due to time zones).

aliok commented on elections/2023-TOC/README.md:
```diff
@@ -0,0 +1,153 @@
+- [Dawn Foster](https://github.com/geekygirldawn)
+- [Ali Ok](https://github.com/aliok)
+
+You can reach us by emailing elections@knative.team
```
Who receives mails to this address? Are we sure that this inbox is working?

geekygirldawn replied:
Ah, good question! That alias is configured in groups.yaml and I've updated it there as another commit within this PR. We'll need to wait until the PR is merged and then test it to make sure it's working.

aliok commented on elections/2023-TOC/README.md:
```diff
@@ -0,0 +1,153 @@
+
+### Decision
+
+Ballots are compiled by Elekto and all candidates are ranked using the Condorcet
```
```suggestion
Ballots are compiled by Elekto and all candidates are ranked using the [Condorcet]
```

aliok commented on elections/2023-TOC/README.md:
```diff
@@ -0,0 +1,153 @@
+[Eligibility for candidacy]: https://github.com/knative/community/blob/master/mechanics/TOC.md#candidate-eligibility
+[Eligibility for voting]: https://github.com/knative/community/blob/master/mechanics/TOC.md#candidate-eligibility#voter-eligibility
+[Elekto]: https://elections.knative.dev
+[file a voting exception]: https://elections.knative.dev/app/elections/2023-TOC/exception
```
```suggestion
[file a voting exception]: https://elections.knative.dev/app/elections/2023-TOC/exception
[Condorcet]: https://en.wikipedia.org/wiki/Condorcet_method
```

I suggested to create a link for Condorcet above.

aliok reviewed (commented):
LGTM

I compared files with Knative TOC 2022 election and Kubernetes SC 2022 election files. Suggested some optional minor changes.

I am curious about the elections@knative.team address though.

All good!

aliok commented on elections/2023-TOC/README.md:
````diff
@@ -0,0 +1,153 @@
+candidate-yourname.md
+```
+
+`yourname` in the template file should match your GitHub handle.
````
I've changed all "GitHub"s to "Github" because I noticed you did it in one location.

geekygirldawn replied:
This was a typo in the docs for the last election. Proper capitalization is "GitHub" with a capital H, so I think I've fixed it in most places.

aliok replied:
I did it the other way around, sorry. (Late night PR review)

Onceagain12 reviewed (approved):
Summit

lance commented on elections/2023-TOC/election_desc.md:
```diff
@@ -0,0 +1,27 @@
+The role of this election is to fill the one (1) seat due for election this year on the [Knative Technical Oversight Committee](https://github.com/knative/community/blob/master/TECH-OVERSIGHT-COMMITTEE.md). The elected member will serve a two (2) year term.
+
+This election will shape the future of Knative as a community and project.
+While WGs help shape the individual direction of the project, the
```
Couple of small nits... "WG" is not spelled out anywhere in this document, so I suggest writing it out fully. Also, "individual direction of the project" is a little unclear to me. Here is a suggestion that may be better.
```suggestion
While working groups help shape the direction of individual subprojects, the
```

lance reviewed (commented):
Looks good overall - thanks for getting this going, @geekygirldawn! I just have a few minor comments

geekygirldawn commented:
I've addressed all of the feedback on this PR, so we can remove the hold and merge the PR as soon as @jberkus confirms that he's pushed the changes to Elekto that we needed.

geekygirldawn commented on groups/committee-steering/groups.yaml:
```diff
@@ -24,4 +24,4 @@ groups:
     members:
       - fosterd@vmware.com
       - jberkus@redhat.com
```
We talked about this and would like to leave Josh on the ML for the election officers in case there are any urgent Elekto issues that come up in Pacific time, since Ali and I are both in European time zones.

geekygirldawn commented:
We've pushed the announcement date for the election to May 3 to give Josh some extra time to set up Elekto and make sure we have everything ready for the announcement.

aliok commented on elections/2023-TOC/README.md:
```diff
@@ -0,0 +1,155 @@
+## Background
+
+This election will shape the future of Knative as a community and project.
+While WGs help shape the individual direction of the project, the
+[TOC Charter] covers the technical direction, health of the project
+and community as a whole.
```
```suggestion
While working groups help shape the direction of individual subprojects, the
[TOC Charter] covers the technical direction, health of the project
and community as a whole.
```

Copy pasted this better text from the election_desc.md file

jberkus commented:
/remove-hold
/lgtm

Ready to merge.

salaboy commented:
/lgtm

salaboy commented:
/approve
